src/SFA.DAS.ApprenticeFeedback.Domain/Models/ApprenticeFeedbackTarget.cs

```diff
@@ -4,12 +4,16 @@ namespace SFA.DAS.ApprenticeFeedback.Domain.Models
 {
     public class ApprenticeFeedbackTarget
     {
-        public Guid? Id { get; set; }
```

Comment:
I think this might need to stay null as it isn't always set, and we don't want to assign a default non guid when inserting records. but worth checking by inserting records and see what happens